dist/src/main/java/io/camunda/application/commons/migration/MigrationProfilePostProcessor.java

```diff
@@ -22,6 +22,9 @@ public void postProcessEnvironment(
     if (environment.acceptsProfiles(Profiles.of(Profile.MIGRATION.getId()))) {
       environment.addActiveProfile(Profile.PROCESS_MIGRATION.getId());
       environment.addActiveProfile(Profile.IDENTITY_MIGRATION.getId());
+    } else if (environment.acceptsProfiles(Profiles.of(Profile.TASKLIST.getId()))) {
```

Comment:
:question: Can we be sure that the post processor is executed when no migration profil is set?

Reply:
Yes this post processor is defined in the [spring.factories](https://github.com/camunda/camunda/blob/d5dd4d107b02d354ff6c863363f001bcf402e0ea/dist/src/main/resources/META-INF/spring.factories) and it should be picked up always when a Spring application is running.